zammad/values.yaml

```diff
@@ -402,6 +402,12 @@ autoWizard:
   #     ]
   #   }
 
+commonLabels: {}
+  # my-label: "value"
+commonAnnotations: {}
+  # my-annotation: "value"
+podLabels: {}
+  # my-label: "value"
 podAnnotations: {}
```

Comment:
imho thats not very flexible as you only can set same annotations on all pods.
should be available per pod.
in the end would be same as commonAnnotations, or not?

Reply:
Should be fixed by a36b0b94cd0ec3a25119f61f54c47ae37360a6c7, can you check please?

@monotek Could you please add my user mgruner as an admin to the zammad organisation on artifacthub?